README.md

```diff
@@ -46,15 +46,16 @@ For incremental builds, the `rush build` command can be used to only build packa
 
 1. Make source code changes on a new Git branch
 2. Ensure unit tests pass when run locally: `rush test`
-3. Locally commit changes: `git commit` (or use the Visual Studio Code user interface)
-4. Repeat steps 1-3 until ready to push changes
-5. Add changelog entry (which could potentially cover several commits): `rush change`
-6. Follow prompts to enter a change description or press ENTER if the change does not warrant a changelog entry. If multiple packages have changed, multiple sets of prompts will be presented. If the changes are only to non-published packages (like **display-test-app**), then `rush change` will indicate that a changelog entry is not needed.
-7. Completing the `rush change` prompts will cause new changelog entry JSON files to be created.
-8. Check for API signature changes: `rush extract-api`.  This will update the signature files, located in `common/api`.
-9. Review any diffs to the API signature files in the `common/api` directory to ensure they are compatible with the intended release of the package
-10. Commit the changelog JSON files and any API signature updates.
-11. Publish changes on the branch and open a pull request.
+3. Ensure linting passes when run locally: `rush lint`
+4. Locally commit changes: `git commit` (or use the Visual Studio Code user interface)
+5. Repeat steps 1-3 until ready to push changes
+6. Add changelog entry (which could potentially cover several commits): `rush change`
+7. Follow prompts to enter a change description or press ENTER if the change does not warrant a changelog entry. If multiple packages have changed, multiple sets of prompts will be presented. If the changes are only to non-published packages (like **display-test-app**), then `rush change` will indicate that a changelog entry is not needed.
+8. Completing the `rush change` prompts will cause new changelog entry JSON files to be created.

```

Comment:
rushChange.bat does `git commit -a --amend --no-edit` for you. `rush change` requires you to do it explicitly - should document that here?

Reply:
Yes, we should document it. I'm going to switch around some of this workflow as the way it's described now it does require two separate commits or adding more than just changelogs with an amend.

Reply:
Yeah, `rush change` seems like it ought to be the last commit you make.

Reply:
Scratch that, the edit workflow mentions making multiple commits in step 4 and if that's the case I don't really see a need to amend for the changelogs. Especially since we squash all PRs.

I am going to switch from `rush test` -> `rush cover` as you could hit test coverage minimums that would fail the build.

Reply:
Switched around to put `rush change` as the last set of steps, after `rush extract-api`.  Let me know what you think.